README.md

```diff
@@ -1,94 +1,206 @@
+cached in FIFO (~1000 samples).
+
+- 50 timestamps on U64ResponseFIFO.
+- 50 * 3 * [number of accelerometers] double values (raw accelerometer values) on SQLResponseFIFO.
+
+### Response to command 3
+- 1 U64 value on U64ResponseFIFO, representing chassis temperature in C * 100.
+
+Chassis temperature is measured with 0.25 degC resolution.
+
+## cRIO modules settings
+
+NI-9234 and NI-9232 should be configured for calibrated mode. The DIO modules
+clocks on single cRIO shall be synchronized to single time source - currently
+the first DIO module, in future NI-9469 sync. This is required to read all DIOs
+at the same time.
+
+## Timestamp synchronization
+
+Timing of retrieved data, and particularly synchronization among different data
+stream, is critical for this application. With incorrect time, data cannot be
+easily correlated among different sources.
+
+cRIOs are equipped with NI-9469 synchronization module. One of the VMS cRIOs
+is designated as source of the synchronization signals, generating triggers at
+regular intervals. This source cRIO is refered to as controller in the
+following paragraphs. The other cRIOs are designated as responders. Responders
+receive clock triggers from the controller. NI-9469 provides 4 trigger lines,
+which are used as following:
+
+| Line | Used for                                                                                                                       |
+| ---- | ------------------------------------------------------------------------------------------------------------------------------ |
+| 0    | Triggered for controller-responder synchronization. Data shall be avereged out, base clock increased when this trigger occurs. |
+| 1    | Send from controller to responder to zero base clocks, start accelometer collecting.                                           |
+| 2    | Not used.                                                                                                                      |
+| 3    | Not used.                                                                                                                      |
+
+To synchronize clocks, master shall establish (and re-establish) base clocks.
+Base clocks offset to TAI is then published in the "baseClockOffset" field of
+the "timeSynchronization" event.
+
+### Time synchronization state machine
+
+CSC time synchronization state is held in "state" field of the
+"timeSynchronization" event.
+
+```mermaid
+flowchart TD
+  Initialising--Controller-->CReady[Controller Ready]
+  Initialising--Responder-->RWaiting[Responder Waiting]
+  subgraph controller
+  CReady-->CRunning[Controller Running]
+  end
+  subgraph responder
+  RWaiting-->RSynced[Responder Synced]
+  end
+  controller-->Fail[Synchronization Failed]
+  responder-->Fail[Synchronization Failed]
+  subgraph init[" "]
+  Fail-->Initialising
+  end
+```
+
+### Initialising state
+
+When CSC is started, it signals in its state it is initialising. In this state,

```

Comment:
"it signals in its state it is initialising" --> I do not understand this sentence.

Reply:
CSC is started in initialising state.